csi.proto

```diff
@@ -1020,6 +1023,54 @@ message NodeUnpublishVolumeRequest {
 message NodeUnpublishVolumeResponse {
   // Intentionally empty.
 }
+message NodeGetVolumeStatsRequest {
+  // The ID of the volume. This field is REQUIRED.
+  string volume_id = 1;
+
+  // The path at which the volume was published. It MUST be an absolute
+  // path in the root filesystem of the process serving this request.
+  // This is a REQUIRED field.
+  string target_path = 2;
+}
+
+message NodeGetVolumeStatsResponse {
```

Comment:
What about consolidating this a bit w/ units that we can extend later?
```protobuf
message Usage {
  enum Unit {
    UNKNOWN = 0;
    INODE = 1;
    CLUSTER = 2;
  }

  int64 free = 1;
  int64 total = 2;
  Unit unit = 3;
} 

repeated Usage usage = 1;
```

Reply:
fixed